src/nimibook/pubs.nim

```diff
@@ -1,7 +1,7 @@
 
```

Comment:
Nitpicks: blankline here and imho the filename is better as `publishers.nim`.

Reply:
done!